ibm/data_source_ibm_cis_waf_groups.go

```diff
@@ -0,0 +1,117 @@
+package ibm
+
+import (
+	"log"
+
+	"github.com/IBM/go-sdk-core/v3/core"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const cisWAFGroups = "waf_groups"
+
+func dataSourceIBMCISWAFGroups() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIBMCISWAFGroupsRead,
+		Schema: map[string]*schema.Schema{
+			cisID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "CIS Intance CRN",
+			},
+			cisDomainID: {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "CIS Domain ID",
+				DiffSuppressFunc: suppressDomainIDDiff,
+			},
+			cisWAFGroupPackageID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "WAF Rule package id",
```

Comment:
No Diffsupress added here..

Reply:
already added for package_id.. added for group_id.